hazelcast/src/main/java/com/hazelcast/internal/util/StringUtil.java

```diff
@@ -57,28 +57,11 @@ public final class StringUtil {
     public static final Pattern VERSION_PATTERN
             = Pattern.compile("^(\\d+)\\.(\\d+)(\\.(\\d+))?(-\\w+(?:-\\d+)?)?(-SNAPSHOT)?$");
 
-    /**
-     * Empty String.
-     */
-    public static final String EMPTY_STRING = "";
-
     private static final String GETTER_PREFIX = "get";
 
     private StringUtil() {
     }
 
-    /**
-     * Creates a UTF8_CHARSET string from a byte array.
-     *
-     * @param bytes  the byte array.
-     * @param offset the index of the first byte to decode
-     * @param length the number of bytes to decode
-     * @return the string created from the byte array.
-     */
-    public static String bytesToString(byte[] bytes, int offset, int length) {
```

Comment:
What about `public static String bytesToString(byte[] bytes)`?
Are you planning to remove it either? 

In a code base, we have a mix of usage - `bytesToString(...)` and  `new String(...)` used both.

Reply:
I will try to remove that one too. But in a new PR to keep it small. All methods that delegate to JDK code can be removed in my opinion